Gemfile

```diff
@@ -11,6 +11,8 @@ gem 'jquery-ui-rails', rails_version[0] == '3' ? '~> 4.0' : '~> 5.0'
 
 gem 'test-unit', '~> 3.0' if rails_version[0] == '3'
 
+gem 'arbre', github: 'activeadmin/arbre'
```

Comment:
This is no longer necessary, the change is released in `1.1.1`, but Arbre `1.1.1` should be the new min version in `activeadmin.gemfile`.